kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -65,6 +68,8 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
     if (session == null) {
       throw KyuubiSQLException(s"Invalid $sessionHandle")
     }
+    info(s"Session with $sessionHandle is closed," +
```

Comment:
$sessionHandle is closed, current opening sessions ...

Reply:
I will fix it.